htdocs/loan/class/loan.class.php

```diff
@@ -66,6 +66,7 @@ class Loan extends CommonObject
 	public $account_capital;
 	public $account_insurance;
 	public $account_interest;
+	public $total_paid;
```

Comment:
This field does not exists into database so no reason to have it into class.
Poblem may be in parent code that use it, probably useless.